features/support/data.js

```diff
@@ -9,17 +9,27 @@ const classes = require('./data_classes');
 const tableDiff = require('../lib/table_diff');
 const ensureDecimal = require('../lib/utils').ensureDecimal;
 const errorReason = require('../lib/utils').errorReason;
+const CheapRuler = require('cheap-ruler');
 
 module.exports = function () {
     this.setGridSize = (meters) => {
+        this.gridSize = parseFloat(meters);
+
         // the constant is calculated (with BigDecimal as: 1.0/(DEG_TO_RAD*EARTH_RADIUS_IN_METERS
         // see ApproximateDistance() in ExtractorStructs.h
         // it's only accurate when measuring along the equator, or going exactly north-south
-        this.zoom = parseFloat(meters) * 0.8990679362704610899694577444566908445396483347536032203503E-5;
+        this.zoom = this.gridSize * 0.8990679362704610899694577444566908445396483347536032203503E-5;
```

Comment:
It is now only used in `findNodeByLocation` and we probably could use just `this.gridSize` there, but it is easier to keep it as is.